src/helpers/APIRequester.js

```diff
@@ -0,0 +1,72 @@
+const BASE_URL = "https://boolean-api-server.fly.dev/andreSturesson/";
+
+export async function fetchContacts() {
+  return await fetch(BASE_URL + "contact")
+    .then((response) => response.json())
+    .then((data) => {
+      return data;
+    })
+    .catch((error) => {
+      console.error(error);
+    });
+}
+
+/**
+ * Creates a new contact.
+ * @param {Object} payload - The contact data.
+ * @param {string} payload.firstName - The first name of the contact.
+ * @param {string} payload.lastName - The last name of the contact.
+ * @param {string} payload.email - The email of the contact.
+ * @param {string} payload.street - The street address of the contact.
+ * @param {string} payload.city - The city of the contact.
+ * @param {number} payload.latitude - The latitude of the contact's location.
+ * @param {number} payload.longitude - The longitude of the contact's location.
+ * @param {string} payload.favouriteColour - The favourite color of the contact.
+ * @param {string} payload.profileImage - The URL of the contact's profile image.
+ */
+export async function createContact(payload) {
```

Comment:
you might want to pass a callback for success and a callback for error so that your code can detect if the operation was successful or not:

```js
createContact(payload, onSuccess, onError) {
await fetch(BASE_URL + "contact", {
    method: "POST",
    headers: {
      "Content-Type": "application/json",
    },
    body: JSON.stringify(payload),
  })
    .then((response) => response.json())
    .then(onSuccess)
    .catch(onError)
}

const onSuccess(data) => {... }

const onError = (err) =>  {...}